src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/BasePreRenderer.js

```diff
@@ -14,22 +14,30 @@ const propTypes = {
     /** Press out handler for the code block */
     onPressOut: PropTypes.func,
 
+    /** Styles for the Pre Renderer container */
+    containerStyle: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.object),
+        PropTypes.object,
+    ]),
+
     ...htmlRendererPropTypes,
 };
 
 const defaultProps = {
     onPressIn: undefined,
     onPressOut: undefined,
+    containerStyle: [],
 };
 
 const BasePreRenderer = forwardRef((props, ref) => {
     const TDefaultRenderer = props.TDefaultRenderer;
-    const defaultRendererProps = _.omit(props, ['TDefaultRenderer', 'onPressIn', 'onPressOut', 'onLongPress']);
+    const defaultRendererProps = _.omit(props, ['TDefaultRenderer', 'onPressIn', 'onPressOut', 'onLongPress', 'containerStyle']);
 
     return (
         <ScrollView
             ref={ref}
             horizontal
+            style={props.containerStyle}
```

Comment:
If we're adding `styles.mv2` both in index and index.native, is there anything preventing us from just adding it here instead?

```suggestion
            style={styles.mv2}
```

Reply:
Fixed.